bigquery/main.go

```diff
@@ -134,23 +167,104 @@ func (n *bqNotifier) SendNotification(ctx context.Context, build *cbpb.Build) er
 		log.Warningf("Build passes filter but does not have trigger ID: %v, status: %v", n.filter, build.GetStatus())
 	}
 	log.Infof("sending Big Query write for build %q (status: %q)", build.Id, build.Status)
+	if build.ProjectId == "" {
+		return fmt.Errorf("Build missing project id")
+	}
+	if build.Id == "" {
+		return fmt.Errorf("Build missing id")
+	}
+	newRow := bqRow{}
+	newRow.ProjectID = build.ProjectId
+	newRow.ID = build.Id
+	newRow.BuildTriggerID = build.BuildTriggerId
+	newRow.Status = build.Status.String()
+	if newRow.Status == "STATUS_UNKNOWN" {
+		return fmt.Errorf("Build %v has unknown status", newRow.ID)
+	}
```

Comment:
Nit: prefer to collect all of the data you need first and then assign it all in one shot: 

```go
newRow :=&bqRow{
  ProjectId: build.ProjectId,
  ...
}
```

That way, your code is less prone to drift-related bugs and it's also more readable/clear which value in the row comes from what other object.